tekton/tasks/run-sanity-test-task.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-basic-sanity-check
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: project
+        mountPath: /workspace/source/image
+      - name: directory
+        mountPath: /workspace/source/directory
+  results:
+    - name: testResult
+      description: the result of sanity test
+  steps:
+    - name: clone-git-repo
+      image: quay.io/jsztuka/getgit:v1
+      script: "git clone https://github.com/dirgim/hacbs-conftest.git /project"
+      volumeMounts:
+        - name: project
+          mountPath: /project
+    - name: run-sanity-signature-check-pvc
+      image: quay.io/hongweiliu/conftest
+      script: |
+        #!/bin/sh
+        conftest test /directory/rpm-manifest.json --policy /project/rpm_manifest/policy/unsigned-rpms.rego --output=json > /directory/test-result-in-pvc.json
+      volumeMounts:
+        - name: project
+          mountPath: /project
+        - name: directory
+          mountPath: /directory
+    - name: run-sanity-signature-check-result
+      image: quay.io/hongweiliu/conftest
+      script: |
+        #!/bin/sh
+        conftest test /directory/rpm-manifest.json --policy /project/rpm_manifest/policy/unsigned-rpms.rego --output=json | tee  /tekton/results/testResult && exit 0
+      volumeMounts:
+        - name: project
+          mountPath: /project
+        - name: directory
+          mountPath: /directory
+    - name: show-result-in-this-task
```

Comment:
Results can be displayed in conftest step itself, I think there is no need to have extra step to display results.

Reply:
I think that the point of this step is to display usage of tekton results vs PVCs in the pipeline - @hongweiliu17 do you agree?

Reply:
Great, makes sense now.